Frontend/src/app/components/user-post/user-post.component.html

```diff
@@ -11,9 +11,48 @@
         <span>{{ likes }} Likes</span>
       </div>
       <div class="post-details-row">
-        <a role="button" tabindex="0" (click)="openComments()">
+        <a
+          *ngIf="comments > 1 || comments === 0"
+          role="button"
+          tabindex="0"
+          (click)="openComments()"
+        >
           {{ comments }} Comments
         </a>
+        <a
+          *ngIf="comments === 1"
+          role="button"
+          tabindex="0"
+          (click)="openComments()"
+        >
+          {{ comments }} Comment
+        </a>
```

Comment:
I think you can change `{{ comments }} Comment` to `{{ comments }} {{ comments === 1 ? "Comment" : "Comments" }}` And then remove the ngIf and the other anchor since the only difference is the text